vufs/src/main/java/com/stnetix/ariaddna/vufs/BusinessObjects/MetaTable.java

```diff
@@ -0,0 +1,55 @@
+package com.stnetix.ariaddna.vufs.BusinessObjects;
+
+import com.stnetix.ariaddna.commonutils.mavenutil.MavenUtil;
+
+import java.util.Set;
+import java.util.concurrent.CopyOnWriteArraySet;
+
+/**
+ * Metatable it's kernel structure of VUFS and implementing as acyclic graph.
+ */
+public class MetaTable {
+
+    private String mVersion;
+    private MasterTableType mType;
+    private Set<String> mMetafileUuidSet;
```

Comment:
Oh... I mistake in specification metafiles uuid [] - type: set<metafile>
And rename metafiles uuid [] to metafiles []